pep/app/pods/admin/general/controller.ts

```diff
@@ -49,6 +50,64 @@ export default class AdminGeneral extends Controller {
         });
     }
 
+    /**
+     * Open Video Modal
+     *
+     * @memberof AdminGeneral
+     */
+    @action
+    openVideoModal() {
+        const changeset = createChangeset({}, CONFIGURATION_VIDEO_VALIDATIONS);
+        this.modal.open('admin/video', {
+            changeset,
+            actions: {
+                onSave: this.createVideo
+            }
+        });
+    }
+
+    /**
+     * Open topical video modal
+     *
+     * @memberof AdminGeneral
+     */
+    @action
+    openTopicalVideoModal() {
+        const changeset = createChangeset({}, CONFIGURATION_VIDEO_VALIDATIONS);
+        this.modal.open('admin/video', {
+            changeset,
+            actions: {
+                onSave: this.createTopicalVideo
+            }
+        });
+    }
+
+    /**
+     * Create the topical lvideo object and add it to the changeset
+     *
+     * @param {VideoConfiguration} video
+     * @memberof AdminGeneral
+     */
+    @action
+    createTopicalVideo(video: VideoConfiguration): void {
+        const previews = this.changeset?.get('configSettings.global.cards.topicalVideoPreviews');
+        previews.push(video);
+        this.changeset?.set('configSettings.global.cards.topicalVideoPreviews', [...previews]);
```

Comment:
```suggestion
        this.changeset?.set('configSettings.global.cards.topicalVideoPreviews', [...previews, video]);
```
I think this is a little cleaner for imutable array pushing, nothing to hold up the PR tho.